### PR TITLE
Add password reset

### PR DESCRIPTION
**Pull request by trunghieu0211**

### CODE REVIEW

**Comment by trunghieu0211** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -62,6 +62,20 @@ def send_activation_email
...
+    update_columns reset_digest: User.digest(reset_token),
+     reset_sent_at: Time.zone.now
+  end
+
+  def send_password_reset_email
+    UserMailer.password_reset(self).deliver_now
+  end
+
+  def password_reset_expired?
+    reset_sent_at < 2.hours.ago
```
Comment (huongnguyenmta) [inline app/models/user.rb]: move `2` into `Settings`

**Comment by huongnguyenmta** [comment]
commented

**Comment by trunghieu0211** [comment]
ready

**Comment by huongnguyenmta** [comment]
commented-(merged) thôi bấy bỳ (dance2)-no

**Pull request merged**